runtime/lang/menu_ja_jp.utf-8.vim

```diff
@@ -1,315 +1,1266 @@
-menutrans Sp&lit-Open\.\.\.<Tab>:sp	分割して開く(&L)\.\.\.<Tab>:sp
-menutrans Open\ Tab\.\.\.<Tab>:tabnew	タブページで開く<Tab>:tabnew
-menutrans &New<Tab>:enew		新規作成(&N)<Tab>:enew
-menutrans &Close<Tab>:close		閉じる(&C)<Tab>:close
-menutrans &Save<Tab>:w			保存(&S)<Tab>:w
-menutrans Save\ &As\.\.\.<Tab>:sav	名前を付けて保存(&A)\.\.\.<Tab>:sav
-menutrans Split\ &Diff\ with\.\.\.	差分表示(&D)\.\.\.
-menutrans Split\ Patched\ &By\.\.\.	パッチ結果を表示(&B)\.\.\.
-menutrans &Print			印刷(&P)
-menutrans Sa&ve-Exit<Tab>:wqa		保存して終了(&V)<Tab>:wqa
-menutrans E&xit<Tab>:qa			終了(&X)<Tab>:qa
+menutrans &File ファイル(&F)
+" File menuitems {{{1
+menutrans &Open\.\.\.<Tab>:e 開く(&O)\.\.\.<Tab>:e
+menutrans Sp&lit-Open\.\.\.<Tab>:sp 分割して開く(&L)\.\.\.<Tab>:sp
+menutrans Open\ Tab\.\.\.<Tab>:tabnew タブページで開く<Tab>:tabnew
+menutrans &New<Tab>:enew 新規作成(&N)<Tab>:enew
+menutrans &Close<Tab>:close 閉じる(&C)<Tab>:close
+menutrans &Save<Tab>:w 保存(&S)<Tab>:w
+menutrans Save\ &As\.\.\.<Tab>:sav 名前を付けて保存(&A)\.\.\.<Tab>:sav
+menutrans Split\ &Diff\ With\.\.\. 差分表示(&D)\.\.\.
+menutrans Split\ Patched\ &By\.\.\. パッチ結果を表示(&B)\.\.\.
+menutrans &Print 印刷(&P)
+menutrans Sa&ve-Exit<Tab>:wqa 保存して終了(&V)<Tab>:wqa
+menutrans E&xit<Tab>:qa 終了(&X)<Tab>:qa
+" }}}
 
 " Edit menu
-menutrans &Edit				編集(&E)
-menutrans &Undo<Tab>u			取り消す(&U)<Tab>u
-menutrans &Redo<Tab>^R			もう一度やる(&R)<Tab>^R
-menutrans Rep&eat<Tab>\.		繰り返す(&E)<Tab>\.
-menutrans Cu&t<Tab>"+x			切り取り(&T)<Tab>"+x
-menutrans &Copy<Tab>"+y			コピー(&C)<Tab>"+y
-menutrans &Paste<Tab>"+gP		貼り付け(&P)<Tab>"+gP
-menutrans Put\ &Before<Tab>[p		前に貼る(&B)<Tab>[p
-menutrans Put\ &After<Tab>]p		後に貼る(&A)<Tab>]p
-menutrans &Delete<Tab>x			消す(&D)<Tab>x
-menutrans &Select\ All<Tab>ggVG		全て選択(&S)<Tab>ggVG
-menutrans &Find\.\.\.			検索(&F)\.\.\.
-menutrans &Find<Tab>/			検索(&F)<Tab>/
-menutrans Find\ and\ Rep&lace\.\.\.	置換(&L)\.\.\.
-menutrans Find\ and\ Rep&lace<Tab>:%s	置換(&L)<Tab>:%s
-menutrans Find\ and\ Rep&lace<Tab>:s	置換(&L)<Tab>:s
-"menutrans Options\.\.\.			オプション(&O)\.\.\.
-menutrans Settings\ &Window		設定ウィンドウ(&W)
-menutrans Startup\ &Settings		起動時の設定(&S)
+menutrans &Edit 編集(&E)
+" Edit menuitems {{{1
+menutrans &Undo<Tab>u 取り消す(&U)<Tab>u
+menutrans &Redo<Tab>^R もう一度やる(&R)<Tab>^R
+menutrans Rep&eat<Tab>\. 繰り返す(&E)<Tab>\.
+menutrans Cu&t<Tab>"+x 切り取り(&T)<Tab>"+x
+menutrans &Copy<Tab>"+y コピー(&C)<Tab>"+y
+menutrans &Paste<Tab>"+gP 貼り付け(&P)<Tab>"+gP
+menutrans Put\ &Before<Tab>[p 前に貼る(&B)<Tab>[p
+menutrans Put\ &After<Tab>]p 後に貼る(&A)<Tab>]p
+menutrans &Delete<Tab>x 消す(&D)<Tab>x
+menutrans &Select\ All<Tab>ggVG 全て選択(&S)<Tab>ggVG
+menutrans &Find\.\.\. 検索(&F)\.\.\.
+menutrans Find\ and\ Rep&lace\.\.\. 置換(&L)\.\.\.
+menutrans &Find<Tab>/ 検索(&F)<Tab>/
+menutrans Find\ and\ Rep&lace<Tab>:%s 置換(&L)<Tab>:%s
+menutrans Find\ and\ Rep&lace<Tab>:s 置換(&L)<Tab>:s
+menutrans Settings\ &Window 設定ウィンドウ(&W)
+menutrans Startup\ &Settings 起動時の設定(&S)
 
 " Edit/Global Settings
-menutrans &Global\ Settings		全体設定(&G)
-menutrans Toggle\ Pattern\ &Highlight<Tab>:set\ hls!
-	\	パターン強調切替(&H)<Tab>:set\ hls!
```

Comment:
Your script doesn't seem to handle line concatenation correctly.

Reply:
> Your script doesn't seem to handle line concatenation correctly.

Thanks. Yes, my script can't handle line concatenation. What about add the translations after line concatenation manually? It won't appear after applying this template.